### ISSUE TITLE
Installation issue (diagnostic ID 3B70B681-E0EA-4AFF-9ADB-4C247D09243C)

### ISSUE THREAD

**Issue by rachael** [docker/for-mac#1316]
Docker asks for privileged access to install its networking components and links to the Docker apps which is granted to it. It then fails with the error `Fatal Error: Communication with networking components failed`.

Previous versions of Docker have been installed via CLI (and since removed). The Docker Beta was also installed.

Diagnostic ID 3B70B681-E0EA-4AFF-9ADB-4C247D09243C

Here is a sample of the error output:
```
Docker for Mac: version: 1.13.1 (94675c5a7)
macOS: version 10.11.6 (build: 15G31)
logs: /tmp/3B70B681-E0EA-4AFF-9ADB-4C247D09243C/20170220-123852.tar.gz
failure: Docker.qcow2 missing: the VM has never been started
[ERROR]  vmnetd
         /var/tmp/com.docker.vmnetd.socket does not exist
         Unexpected error (No such file or directory) connecting to /var/tmp/com.docker.vmnetd.socket
         /private/tmp/vmnetd/com.docker.vmnetd is not running
[OK]     dns
[ERROR]  driver.amd64-linux
         com.docker.driver.amd64-linux -db is not running
[OK]     virtualization VT-X
[OK]     app
[ERROR]  moby
         /Users/rae/Library/Containers/com.docker.docker/Data/com.docker.driver.amd64-linux/console-ring does not exist
[OK]     system
[OK]     moby-syslog
[ERROR]  db
         /Users/rae/Library/Containers/com.docker.docker/Data/s40 does not exist
         Unexpected error (No such file or directory) connecting to /Users/rae/Library/Containers/com.docker.docker/Data/s40
         com.docker.db is not running
[OK]     env
[OK]     virtualization kern.hv_support
[ERROR]  slirp
         Unexpected error ((Failure
  "Error connecting socket to 9p endpoint unix:/Users/rae/Library/Containers/com.docker.docker/Data/s51: Unix.Unix_error(Unix.ENOENT, \"connect\", \"\")")) connecting to /Users/rae/Library/Containers/com.docker.docker/Data/s51
         com.docker.slirp is not running
         slirp check failed with: (Failure
  "Error connecting socket to 9p endpoint unix:/Users/rae/Library/Containers/com.docker.docker/Data/s52: Unix.Unix_error(Unix.ENOENT, \"connect\", \"\")")
[ERROR]  osxfs
         com.docker.osxfs is not running
[OK]     moby-console
[OK]     logs
[ERROR]  docker-cli
         cannot find docker
         docker-cli check failed with: (Failure "docker -v: timeout after 10.00s")
[OK]     menubar
[ERROR]  disk
         Docker.qcow2 missing: the VM has never been started
```

Help greatly appreciated! Things I have tried:

1. Deleting `/Library/PrivilegedHelperTools/com.docker.vmnetd` (#777) 
2. Removing any other Docker installations (there was one installed via Brew) and anything else docker-related
3. Downloading a new copy of Docker for Mac

**Comment by ijc25** [comment]
Hi @rachael, thanks for your report.

Please try the following steps:

```
sudo cp $(mdfind Docker.app)/Contents/Resources/com.docker.vmnetd.plist /Library/LaunchDaemons/com.docker.vmnetd.plist
sudo chmod 600 /Library/LaunchDaemons/com.docker.vmnetd.plist
sudo launchctl load -Fw /Library/LaunchDaemons/com.docker.vmnetd.plist
```

and start Docker for Mac.

May be related to docker/for-mac#61. Closing as a dup of that for now, if the above doesn't help then please do reopen.

**Comment by rachael** [comment]
Ah-hah! For some reason even after the `chmod` when I tried to run the last line I got:

`/Library/LaunchDaemons/com.docker.vmnetd.plist: Path had bad ownership/permissions`

I remember trying this before, now. Last time I was stumped and thought it wouldn't work for me. This time after seeing the `cp` I thought to remove the file and try running the lines again -- and they all ran without error. 

Opened Docker and it started up just fine! Thanks! 🎉
